crates/gossip/src/state.rs

```diff
@@ -66,7 +69,7 @@ impl GossipRoundState {
             session_with_peer,
             started_at: tokio::time::Instant::now(),
             session_id,
-            peer_max_op_data_bytes,
+            peer_max_op_data_bytes: peer_max_op_data_bytes as i32,
```

Comment:
Doesn't this also have to change to an i32?

Reply:
I don't think the config should no, that's just a limit value that shouldn't be set to a negative number. It's being assigned to a value that's being used to track actual usage against that limit. I think this is the right place to cast